openvpn/tunnel/setup_tun_interface.go

```diff
@@ -15,15 +15,16 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package linux
+package tunnel
```

Comment:
I don't think this single file deserves to be a library on its own, especially since we're moving openvpn away from the node repo. Can we leave it here for now and if needed split at a later date?

Reply:
Both ideas is good, but lets do it then we need to reuse